documentation/client_implementation.md

```diff
@@ -1211,6 +1211,7 @@ SegmentMaxAge
 CompactEnabled
 CompactMaxGoroutines
 OptimisticConcurrencyControl
+EncryptionDataAtRest
```

Comment:
Would `EncryptDataAtRest` fit better with the options that are already there? It sounds a bit weird. 😅

Reply:
indeed . wip